test/e2e/page-objects/pages/test-dapp-individual-request.ts

```diff
@@ -0,0 +1,26 @@
+import { Driver } from '../../webdriver/driver';
+
+class TestDappIndividualRequest {
```

Comment:
Why does this need to be a class? 🤔

Reply:
not "needed" per se, but I'm applying page objects, as we do in the rest of the dapps/screens.
I agree in this case could be too much? but still if this page grows (have more divs/btns), we'll already have it implemented, if that makes sense

https://github.com/MetaMask/metamask-extension/tree/main/test/e2e/page-objects/pages